msteams-platform/sbs-meeting-content-bubble.yml

```diff
@@ -158,11 +158,11 @@ items:
 
     1. Select your bot from **Recent resources**.
 
-    1. Select **Channels** in the left pane and select **Microsoft Teams** <img src="~/assets/images/bots/teamsicon.png" alt="Teams icon" width="20"/>.
+    1. Select **Channels** in the left pane and select **Microsoft Teams** :::image type="content" source="~/assets/images/bots/teamsicon.png" alt-text="Screenshot showing the home page.":::.
```

Comment:
For inline icons this is incorrect usage of Markdown syntax.

Reply:
The icon markdown rectified.